djangocms_pageadmin/admin.py

```diff
@@ -186,7 +186,7 @@ def _get_edit_link(self, obj, request, disabled=False):
 
     def _get_duplicate_link(self, obj, request, disabled=False):
         url = reverse(
-            "admin:{app}_{model}_duplicate_content".format(
+            "admin:{app}_{model}_duplicate".format(
```

Comment:
I'm not following the reason for renaming from duplicate_content to duplicate when the url is scoped to the application?

Reply:
The core pagecontent admin has a duplicate. But that is for making a duplicate page. When you introduce versioning that view is borked, and should really be replaced with the view that is created in this pageadmin. 

So that's what I'm doing in this PR.

Reply:
That way the url in the toolbar works as expected as well.